src/main/scala/is/hail/rvd/OrderedRVD.scala

```diff
@@ -752,6 +747,7 @@ object OrderedRVD {
 
   def getKeyInfo(
     typ: OrderedRVDType,
+    partitionKey: Int,
```

Comment:
I think `partitionKey` is a confusing name. Can you add a comment somewhere what this means as well?

Reply:
This is definitely an unclear interface. The only use of the partition key here is to define a middle ground between "the RDD is unordered" and "the RDD is ordered by key". I think the right thing to do is to have the scan return the largest prefix of the key by which the rows are ordered. But I think the only place this is used right now is for reading Bgen, and after we have the index and can skip around to read in sorted order without the local sort, I think `getKeyInfo` can get away with only checking unordered vs. ordered by key, so it didn't seem worth doing the more general thing.

I'm still not happy with this interface, but I guess since it's likely to go away soon I'll just add an explanatory comment for now.